[BUG] ImageStabilization/TimeStamp need update

Hi,

Building openiamgeio with vs2017 on windows failed with following errors, could you have a look? 
this due to libraw removed ImageStabilization by https://github.com/LibRaw/LibRaw/commit/e7e885e67746e6f56a29cae8ebabadae77cc7aa3#diff-0a212125e7f2d7e9caff074c99b2a360 and removed TimeStamp by https://github.com/LibRaw/LibRaw/commit/f0a503dc81319337dcddef00d29ffec123fbfd45, it's very old revision, but openiamgeio still reference them.

Failures:
F:\VCPKG\src\buildtrees\openimageio\src\ease-2.0.8-1be86a4a75\src\raw.imageio\rawinput.cpp(859): error C2039: 'ImageStabilization': is not a member of 'libraw_olympus_makernotes_t'
f:\vcpkg\src\installed\x64-windows\include\libraw\libraw_types.h(416): note: see declaration of 'libraw_olympus_makernotes_t'
F:\VCPKG\src\buildtrees\openimageio\src\ease-2.0.8-1be86a4a75\src\raw.imageio\rawinput.cpp(985): error C2039: 'TimeStamp': is not a member of 'libraw_sony_info_t'
f:\vcpkg\src\installed\x64-windows\include\libraw\libraw_types.h(490): note: see declaration of 'libraw_sony_info_t'

**To Reproduce**
Steps to reproduce the behavior:
1. git clone https://github.com/microsoft/vcpkg
2. ./bootstrap-vcpkg.bat
3. ./vcpkg install openimageio[libraw]

--- a/src/raw.imageio/rawinput.cpp
+++ b/src/raw.imageio/rawinput.cpp
@@ -844,7 +844,7 @@ RawInput::get_makernotes_olympus()
     // unsigned     AFAreas[64];
     MAKERF(AFPointSelected);
     MAKERF(AFResult);
-    MAKERF(ImageStabilization);
+    // MAKERF(ImageStabilization);  Removed after 0.19
     MAKERF(ColorSpace);
 #endif
 #if LIBRAW_VERSION >= LIBRAW_MAKE_VERSION(0, 19, 0)
@@ -970,7 +970,7 @@ RawInput::get_makernotes_sony()
         MAKERF(ElectronicFrontCurtainShutter);
     MAKER(MeteringMode2, 0);
     add(m_make, "DateTime", mn.SonyDateTime);
-    MAKERF(TimeStamp);
+    // MAKERF(TimeStamp);  Removed after 0.19, is in 'other'
     MAKER(ShotNumberSincePowerUp, 0);
 #endif
 }

--- a/testsuite/raw/ref/out-libraw0.18.11.txt
+++ b/testsuite/raw/ref/out-libraw0.18.11.txt
@@ -413,7 +413,7 @@
     Olympus:ColorSpace: 0
     Olympus:EXIF_MaxAp: 2
     Olympus:FocusMode: 0, 0
-    Olympus:ImageStabilization: 0
+    Olympus:ImageStabilization: -1
     Olympus:InternalBodySerial: "4031712008217001               "
     Olympus:LensFormat: 8
     Olympus:LensID: 256

--- a/testsuite/raw/ref/out.txt
+++ b/testsuite/raw/ref/out.txt
@@ -364,7 +364,7 @@
     Olympus:ColorSpace: 0
     Olympus:EXIF_MaxAp: 2
     Olympus:FocusMode: 0, 0
-    Olympus:ImageStabilization: 0
+    Olympus:ImageStabilization: -1
     Olympus:InternalBodySerial: "4031712008217001               "
     Olympus:LensFormat: 8
     Olympus:LensID: 256
@@ -527,7 +527,6 @@
     Sony:ImageStabilization: -1
     Sony:LensID: 40
     Sony:SonyCameraType: 2
-    Sony:TimeStamp: 0, 0, 0, 0, 0, 0
 Comparing "RAW_CANON_EOS_7D.CR2.tif" and "ref/RAW_CANON_EOS_7D.CR2.tif"
 PASS
 Comparing "RAW_NIKON_D3X.NEF.tif" and "ref/RAW_NIKON_D3X.NEF.tif"

